src/main/resources/db/migration/V1__initial_tables.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS SAKSKOMPLEKS
+(
+  id                    CHAR(36)                 NOT NULL,
+  bruker_aktor_id       VARCHAR(32)              NOT NULL,
```

Comment:
Skal vi også lagre fnr? Vi har opplevd at selv om NAV skal bruke aktørid så kommer alt av feilsaker på FNR, så det er utrolig nyttig å kunne slå opp på det fremfor å alltid måtte konvertere..

Reply:
det kan vi godt gjøre, men siden sykmeldingen og søknaden bare har referanse til aktørId, så tenker jeg det er greit for øyeblikket. Mapping til fnr vil jo innebære at vi må kontakte aktørregisteret, og det har jeg holdt utenfor denne pr